README.md

```diff
@@ -17,8 +17,8 @@ Load `@octokit/plugin-request-log` and [`@octokit/core`](https://github.com/octo
 
```

Comment:
Hey @kapoor2427, you nearly got it! But there is one more on line 16 that you need to change. Also could you please comment in the issue #101 so that we can assign you to the issue? I helps us to keep track of who is working on it, and to make sure we give you credit for the change when we merge them from all the open pull requests.

Reply:
#101 @gr2m,  Please assign me the issue.

Reply:
You need to add a comment, something like "I created a review: #111, please review" will do, then we can assign you. Otherwise GitHub does not allow us to

Reply:
@gr2m,  I have created a review: #111, please review.

Reply:
@gr2m,  I have created a review: #111, please review.